docs/ref/models/fields.txt

```diff
@@ -160,12 +197,6 @@ be used for organizational purposes::
         ("unknown", "Unknown"),
     ]
 
-The first element in each tuple is the name to apply to the group. The
-second element is an iterable of 2-tuples, with each 2-tuple containing
-a value and a human-readable name for an option. Grouped options may be
-combined with ungrouped options within a single list (such as the
-``'unknown'`` option in this example).
-
 For each model field that has :attr:`~Field.choices` set, Django will add a
```

Comment:
I think the paragraph below (For each model...) should be moved to after the following one which reads:

> Note that choices can be any sequence object -- not necessarily a list or tuple. 

My point being is that it may be better to group all the paragraphs about choices allowed type and then add any extra information like the display method.

Reply:
Sure, I guess that makes sense.